File: src/main/java/site/gachontable/presentation/admin/dto/request/PubManageRequest.java
package site.gachontable.presentation.admin.dto.request;

import java.util.List;

public record PubManageRequest(List<String> thumbnails,
                               List<MenuRequest> menuRequests) {
    public record MenuRequest(Integer menuId,
                              String thumbnail,
                              String menuName,
                              String price,
                              String oneLiner) {
    }
}

File: src/main/kotlin/site/gachontable/presentation/admin/dto/request/PubManageRequest.kt
package site.gachontable.presentation.admin.dto.request

data class PubManageRequest(
    val thumbnails: MutableList<String>,
    val menuRequests: MutableList<MenuRequest>,
) {
    data class MenuRequest(
        val menuId: Int,
        val thumbnail: String,
        val menuName: String,
        val price: String,
        val oneLiner: String,
    )
}
